src/psyclone/core/single_variable_access_info.py

```diff
@@ -208,39 +216,59 @@ def var_name(self):
         '''
         return str(self._signature)
 
-    def is_written(self):
-        ''':returns: True if this variable is written (at least once).
-        :rtype: bool
+    def is_called(self) -> bool:
+        '''
+        :returns: whether or not any accesses of this variable
+                  represent a call.
+        '''
+        return any(info.access_type == AccessType.CALL for
+                   info in self._accesses)
+
+    def is_queried(self) -> bool:
```

Comment:
To not introduce new terms maybe we should use `is_inquired` and the same in the docstring.

Reply:
But INQUIRY is how Fortran calls it, I am not fond on changing it and making it PSyIR specific. Maybe "is_inquiry" is a good compromise even if it doesn't match the other tenses?

Reply:
OK, I'll go with that.

Reply:
I changed my mind - the only actual use of this method is in a test so I've just removed it entirely.